src/index.js

```diff
@@ -1,27 +1,7 @@
-const express = require('express');
+const colors = require('colors/safe');
 
-// ENV configuration
-require('dotenv').config();
-
-const { contributors, faq, admin } = require('./routes');
-const { server: { port, host } } = require('./config');
-
-// Database connection
-require('./helpers/databaseConnection');
-
-const { isAdmin } = require('./middlewares');
-
-// Initialize server
-const app = express();
-
-// App middleware
-app.use(express.json());
-
-// App routes
-app.get('/', (req, res) => res.json({ msg: 'Check out /contributors and /faq' }));
-app.use('/contributors', contributors);
-app.use('/faq', faq);
-app.use('/admin', isAdmin, admin);
+const { PORT, HOST } = require('./config');
+const app = require('./app');
 
 // Run server
-app.listen(port, host, () => process.stdout.write(`\n\n\x1b[34m Server started @ http://${host}:${port}/\x1b[0m \n`));
+app.listen(PORT, HOST, () => console.log(colors.cyan(`Server started @ http://${HOST}:${PORT}/`)));
```

Comment:
What is the benefit to having the console.logs? process.stdout.write is not much longer and makes the linting response much cleaner? If we want to stick with console.log we should override the rule int he config so as not to have the warnings come up...?